components/server/src/util/log-context.ts

```diff
@@ -32,7 +32,7 @@ LogContext.setAugmenter(augmenter);
 
 export async function runWithContext<T>(
     contextKind: string,
-    context: LogContext & { contextId?: string },
+    context: LogContext & { contextId?: string } & any,
```

Comment:
:cloud: When we talk about tracing, it would also be nice to talk about the shape of this `context`. But out of scope here.